cmd/simulator/txs/agent.go

```diff
@@ -53,33 +56,70 @@ func (a issueNAgent[T]) Execute(ctx context.Context) error {
 	}
 
 	txChan := a.sequence.Chan()
+	confirmedCount := 0
+	batchI := 1
```

Comment:
```suggestion
	batchI := 0
```

Reply:
thanks

Reply:
thought it would be nicer from a user reading the logs to see it 1 -indexed

Reply:
Fair point haha